data/crates.json

```diff
@@ -697,53 +698,49 @@
                         },
                         {
                             "name": "ORMs",
-                            "recommendations": [{
-                                "name": "sea-orm",
-                                "notes": "Recommended. Built on top of sqlx. There is also a related sea-query crate that provides a query builder without full ORM functionality."
-                            }, {
+                            "recommendations": [
+                            {
                                 "name": "diesel",
-                                "notes": "Stable release is sync only. Pre-release async branch is currently AGPL. Can provide better performance in some circumstances."
+                                "notes": "Has excellent performance and takes an approach of strict compile time guarantees. The main crate is Sync only, but <a href=\"https://lib.rs/crates/diesel-async\">diesel-async</a> provides an async connection implementation."
```

Comment:
Nit: I don't think capitalizing "Sync" makes much sense? I would probably phrase it as more like "does not support async/await".